api/src/Security/OAuth/GoogleAuthenticator.php

```diff
@@ -57,7 +57,8 @@ public function authenticate(Request $request): Passport {
                 }
 
                 // do we have a matching user by email?
-                $profile = $profileRepository->findOneBy(['email' => $email]);
+                $profiles = $profileRepository->findBy(['email' => strtolower($email)]);
+                $profile = count($profiles) > 0 ? $profiles[0] : null;
```

Comment:
As long as we don't do the data migration, this will break the login for people who log in via OAuth and have capital letters in their email address on the external identity provider.

We should probably do the data migration, and also make sure that the email address which we get from the OAuth providers ([here](https://github.com/ecamp/ecamp3/blob/devel/api/src/Security/OAuth/GoogleAuthenticator.php#L65) and [here](https://github.com/ecamp/ecamp3/blob/devel/api/src/Security/OAuth/HitobitoAuthenticator.php#L71)) is lower cased.

Another idea I had to go without data migration would be to compare here in the authenticators using SQL `WHERE LOWER("email") = ?`. But that might introduce be a security issue, and would break our undocumented feature which allows users to have both a normal login and an OAuth login to the same account.